virtool/users/data.py

```diff
@@ -150,6 +156,12 @@ async def update(self, user_id: str, data: UpdateUserRequest):
 
         try:
             update["administrator"] = data["administrator"]
```

Comment:
I think you should use `if "administrator" in update`.

The try-except encloses too much code and there is a risk a `KeyError` could bubble up from further down the stack and be ignored.